server/routes/v1/deployment.ts

```diff
@@ -9,9 +9,15 @@ import { BadReq, NotFound, Forbidden } from '../../utils/result'
 import { findModelByUuid } from '../../services/model'
 import { findVersionByName } from '../../services/version'
 import { createDeployment, findDeploymentByUuid, findDeployments } from '../../services/deployment'
+import _ from 'lodash'
+import { Deployment } from '../../../types/interfaces'
 
 const nanoid = customAlphabet('0123456789abcdefghijklmnopqrstuvwxyz', 6)
 
+const deploymentSubset = ((deployment: Deployment) => {
```

Comment:
This type of code should be in the `deployment` service I imagine.  As it'll be common to a lot of things.

Reply:
Now a serializer